site/operational-guide/tasks.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Administrative Tasks
+menu_order: 60
+---
+## Configuring Weave to Start Automatically on Boot
+
+`weave launch` runs all weave's containers with a Docker restart
+policy of `always`, so as long as you have launched weave manually
+once and your system is configured to start Docker on boot then weave
+will be started automatically on system restarts.
+
+If you're aiming for a non-interactive installation, you can use
+systemd to launch weave after Docker - see [systemd docs](/site/systemd.md) for
+details.
+
+## Detecting Lost IP Address Space
+
+The recommended way of removing a peer is to run `weave reset` on that
+peer before the underlying host is decommissioned or repurposed - this
+ensures that the portion of the IPAM allocation range assigned to the
+peer is released for reuse. Under certain circumstances this operation
+may not be successful, or indeed possible:
+
+* If the peer in question is partitioned from the rest of the network
+  when `weave reset` is executed on it
+* If the underlying host is no longer available to execute `weave
+  reset` - for example due to a hardware failure or other unplanned
+  termination
+
+In some cases you may already be aware of the problem, as you were
+unable to execute `weave reset` successfully or because you know
+through other channels that the host has died - in these cases you can
+proceed straight to the section on reclaiming lost space.
+
+However in some scenarios (for example when using autoscaling and
+spot-instances that can be destroyed without notice) it may not be
+obvious that space has been lost, in which case you can check for it
+periodically with the following command on any peer:
+
+    weave status ipam
+
+This will list the names of unreachable peers; if you are satisifed
+that they are truly gone, rather than temporarily unreachable due to a
+partition, you can reclaim their space by following the advice in the
+next section.
+
+## Reclaiming Lost IP Address Space
```

Comment:
I'm pretty sure I split this into two sections at your behest, but I can't prove it :smile:

Combined.